rspace/src/main/scala/coop/rchain/rspace/history/package.scala

```diff
@@ -0,0 +1,191 @@
+package coop.rchain.rspace
+
+import java.lang.{Byte => JByte}
+
+import com.typesafe.scalalogging.Logger
+import coop.rchain.shared.AttemptOps._
+import scodec.Codec
+
+import scala.annotation.tailrec
+
+package object history {
+
+  private val logger: Logger = Logger[this.type]
```

Comment:
It's more convenient to extend `com.typesafe.scalalogging.LazyLogging`

Reply:
Tell me more...

Reply:
Oh interesting - I tend to default to importing and only extend when there is an explicit motivation to do so.  

One question that arises for me with this approach is when bringing something in scope, when do we choose to `extend` a `class`/`object`/`package object` vs when do we simply import.  Seems sorta arbitrary to choose to extend `LazyLogging` but not other things that we need in scope, yah?

Also, vis-a-vis lazy vs strict logging - I understand the motivation for using a lazy logger in a class that is instantiated many times, but here the strict logger is only instantiated once.